.github/workflows/release.yml

```diff
@@ -26,5 +26,10 @@ jobs:
           fetch-depth: 0
       - uses: ./.github/actions/setup
       - uses: ./.github/actions/configure-git
-      - name: Release with Gradle
-        run: ./gradlew clean release -Dorg.gradle.daemon=false --stacktrace -x test -x integrationTest
+      - name: Release with JReleaser
+        run: ./gradlew clean release -Dorg.gradle.daemon=false --stacktrace
```

Comment:
IIUC the release is executed after a successful CI build, so there isn't a need to run all the integration tests again.